api/model/person.js

```diff
@@ -1,5 +1,5 @@
 const Promise = require('bluebird');
-const Bcrypt = require('bcrypt');
```

Comment:
BcryptJS is lighter and in pure JS

Reply:
Is it backward compatible?
Does it require a different node version?

Reply:
I'm also not sure about using this bcyptjs - the [original](https://www.npmjs.com/package/bcryptjs) package was not updated for about 3 years now, and this [@jswebfans/bcryptjs](https://www.npmjs.com/package/@jswebfans/bcryptjs) package is some fork with this single release and without a codebase (all links still refer to the original package source).

And regarding @dortheimer 's comment - maybe we should decide on the project's node version and cement it (.nvmrc or w/e method) so we are all on the same page. Currently it seems the dependencies won't install on node version 12 onwards...

Reply:
@GalGend, can you change this back?